### PR TITLE
[ENH] Added Abagen and NiMARE support.

### PR DESCRIPTION
**Pull request by ReinderVosDeWael**
This PR implements abagen and nimare into BrainStat. To that end, it also impements a new `interpolation` submodule within the mesh module which allows for the interpolation of surface label data to a volume. 

Note that this new interpolation module requires that the package pyembree is installed. This package is only available through conda-forge and is itself dependent on embree. As such, we should probably include separate installation instructions for this in the documentation.

### CODE REVIEW

**Comment by ReinderVosDeWael** [comment]
I'll add neurosynth in this same PR. I've returned this PR to a draft until that's done.

File: brainstat/context/utils.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+import numpy as np
+import tempfile
+import gzip
+import shutil
+from brainspace.mesh.mesh_io import read_surface
+from brainspace.vtk_interface.wrappers.data_object import BSPolyData
+from brainstat.mesh.interpolate import surface_to_volume
+
+
+def mutli_surface_to_volume(
```
Comment (sheyma) [inline brainstat/context/utils.py:14]: maybe changing function name into "multi_surface_to_volume"

**Review by sheyma — COMMENTED** [review]
great PR!
Python CI seems to fail bcs of minor flake8 issues. 
Once failures in Python CI fixed, feel free to merge this PR to master.

File: brainstat/context/utils.py (hunk 1 above)
Comment (ReinderVosDeWael) [reply brainstat/context/utils.py:14]: Change multi_surface_to_volume into multi_surface_to_volume?

**Comment by ReinderVosDeWael** [comment]
Changed. this back to a normal PR. The final list of changes is as follows:

- Added brainstat.mesh.interpolate surface to volume interpolations.
- Implemented abagen.
- Implemented NIMARE, although we may wish to change implementation details at a later date.
- Implemented loading of MPC gradients through enigmatoolbox - this is intended as temporary for the OHBM abstract until we write a better BigBrain module.

File: brainstat/context/utils.py (hunk 1 above)
Comment (sheyma) [reply brainstat/context/utils.py:14]: nope, I meant changing  mu**tl**i_surface_to_volume into mu**lt**i_surface_to_volume

**Pull request merged**